surfer/viz.py

```diff
@@ -361,7 +363,9 @@ class Brain(object):
         views to use
     offset : bool
         If True, aligs origin with medial wall. Useful for viewing inflated
-        surface where hemispheres typically overlap (Default: True)
+        surface where hemispheres typically overlap. For flat patches,
+        this aligns the two patches along their posterior
+        edges. (Default: True)
```

Comment:
You should also update the `surface` argument above to say that it's supported, and add a:
```

.. versionchanged:: 0.10
   Support for flat maps.
```